Example/AuthChallengeViewController.swift

```diff
@@ -0,0 +1,55 @@
+//
+//  AuthChallengeViewController.swift
+//  Example
+//
+//  Created by Bob Obi on 25.10.17.
+//  Copyright © 2017 RxSwift Community. All rights reserved.
+//
+
+import UIKit
+import WebKit
+import RxWebKit
+import RxSwift
+import RxCocoa
+
+class AuthChallengeViewController: UIViewController {
+
+    let bag = DisposeBag()
+    let wkWebView = WKWebView()
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        view.addSubview(wkWebView)
+        wkWebView.load(URLRequest(url: URL(string: "https://jigsaw.w3.org/HTTP/Basic/")!))
+        
+        wkWebView.rx
+            .didReceiveChallenge
+            .debug("didReceiveChallenge")
+            .subscribe(onNext: {(webView, challenge, handler) in
+                guard challenge.previousFailureCount == 0 else {
+                    handler(URLSession.AuthChallengeDisposition.performDefaultHandling, nil)
+                    return
+                }
+                /*
+                 The correct credentials is as follows:
```

Comment:
nit: Credentials *are* as follows
Probably better to write "The correct credentials are:"